### PR TITLE
Bump version and start Changelog for 0.29.0.gfm.11

### PR DESCRIPTION
**Pull request by anticomputer**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+  * Added fuzzing target for bracketed patterns (#318)
+  * Fixed bug in list numbering introduced in
+  763587e8775350b8cb4a2aa0f4cec3685aa96e8b (#322) which caused list numbers
+  to increment by 2
+  * Fixed strict prototype clang warning (#310)
+  * Fixed regression test (#312)
+  
+  Note: these changes remove redundant bold tag nesting which may result
+  in existing rendering tests failing, e.g. rendering "____bold____" to html
+  will no longer yield "<p><strong><strong>bold</strong></strong></p>".
```
Comment (kevinbackhouse) [inline changelog.txt:15]: This was already mentioned under 0.29.0.gfm.10

Comment (anticomputer) [reply changelog.txt:15]:
```suggestion
```

**Review by phillmv — APPROVED** [review]

**Review by kevinbackhouse — APPROVED** [review]

**Comment by waldyrious** [comment]
Shouldn't this change have been included in https://github.com/github/cmark-gfm/releases/tag/0.29.0.gfm.11?

**Comment by anticomputer** [comment]
> Shouldn't this change have been included in https://github.com/github/cmark-gfm/releases/tag/0.29.0.gfm.11?

Thanks for catching this @waldyrious, we just re-released .11

**Pull request merged**